nvflare/private/fed/server/job_cmds.py

```diff
@@ -38,16 +40,10 @@ def get_spec(self):
             name="job_mgmt",
             cmd_specs=[
                 CommandSpec(
-                    name="list_all_jobs",
-                    description="list all job defs",
-                    usage="list_all_jobs",
-                    handler_func=self.list_all_jobs,
-                ),
-                CommandSpec(
-                    name="get_job_details",
-                    description="get the details for a job",
-                    usage="get_job_details job_id",
-                    handler_func=self.get_job_details,
+                    name="list_jobs",
```

Comment:
We need the "authz_func" for the "list_jobs", "delete_job" and "clone_job" commands. Currently only "abort_job" command has it.

Reply:
Will tackle this in next PR.